src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -425,44 +432,62 @@ Environment variable: `$WEB_EXT_API_SECRET`
 
 #### `--api-url-prefix`
 
-The signing API URL prefix. This should always be a string. If not specified, this will default to&nbsp;`https://addons.mozilla.org/api/v4` which is the production API.
+The signing API URL prefix. This should always be a string. If not specified, this will default to&nbsp;`https://addons.mozilla.org/api/v4` which is the production API.  
+
+::: note alert
+This variable is not used with the add-on submission API. See --amo-base-url instead.
+:::
 
 Environment variable: `$WEB_EXT_API_URL_PREFIX`
 
+#### `--amo-base-url`
+
+The add-on submission API base URL. This should always be a string. If not specified, this will default to&nbsp;`https://addons.mozilla.org/api/v5` which is the production API.  
+
+::: note alert
+This variable is not used with the signing API. See --api-url-prefix instead.
+:::
+
+Environment variable: `$WEB_EXT_AMO_BASE_URL`
+
 <section id="api-proxy"></section>
 
 #### `--api-proxy`
 
-A proxy host to use for all API connections. Example: `https://yourproxy:6000.`Read more about [how proxy requests work](https://github.com/request/request#proxies). There is a separate section about [signing in a restricted environment](/documentation/develop/getting-started-with-web-ext/#restricted-environment) if the proxy approach doesn't work for you.
+A proxy host to use for all API connections. Example: `https://yourproxy:6000.`Read more about [how proxy requests work](https://github.com/request/request#proxies). There is a separate section about [signing in a restricted environment](/documentation/develop/getting-started-with-web-ext/#restricted-environment) if the proxy approach doesn't work for you. 
```

Comment:
So, these extra spaces are because I originally wrote the "note alert"s as a follow on sentence and apparently stripping off spaces on save is a per-project rather than a global setting in vscode 😬